lib/gds_api/test_helpers/account_api.rb

```diff
@@ -207,6 +207,140 @@ def stub_account_api_request(method, path, with: {}, response_status: 200, respo
           stub_request(method, "#{ACCOUNT_API_ENDPOINT}#{path}").with(**with).to_return(**to_return)
         end
       end
+
+      ######################
+      # GET /api/saved_pages
+      ######################
+      def stub_account_api_has_saved_pages(**options)
+        stub_account_api_request(
+          :get,
+          "/api/saved_pages",
+          response_body: { saved_pages: [{ page_path: "/foo" }, { page_path: "/bar" }] },
+          **options,
+        )
+      end
```

Comment:
This should take the saved pages to return as a parameter, like the `values` param in

```ruby
      def stub_account_api_has_attributes(attributes: [], values: {}, **options)
        querystring = Rack::Utils.build_nested_query({ attributes: attributes }.compact)
        stub_account_api_request(
          :get,
          "/api/attributes?#{querystring}",
          response_body: { values: values },
          **options,
        )
      end
```

Then `stub_account_api_does_not_have_saved_pages` isn't needed any more

Reply:
sure, i renamed it to `stub_account_api_returning_saved_pages`, as I thought otherwise the naming with `has` and returning something empty might be unexepected.